lib/srfi/%3a176/version-flag.sls

```diff
@@ -0,0 +1,16 @@
+(library (srfi :176 version-flag)
+         (export version-alist)
+         (import (rnrs) (only (mosh config) get-config))
+
+(define (version-alist)
+  `((version ,(get-config "version"))
+      (command "mosh")
+      (scheme.id mosh)
+      (install-dir ,(string-append (get-config "prefix") "/bin"))
+      (languages scheme r6rs)
+      (encodings utf-8)
+      (cache-path ,(get-config "mosh-cache-dir"))
+      (website "https://mosh.monaos.org/")
+      (scheme.path ,(get-config "library-path"))
```

Comment:
Does `(get-config "library-path")` always return only one directory, or can it return multiple directories?

Reply:
It returns only one directory(=Default library path).

Reply:
What happens if you set `MOSH_LOADPATH` to more than one directory?